CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Why change this? A lot of systems only have older versions. 2.8 is very common. Is there some new feature that's a must?